.github/workflows/workflow.yml

```
@@ -27,21 +26,20 @@ jobs:
             os: windows-latest
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

This might need
 
```suggestion
          fetch-depth: 0
          ref: ${{ github.ref }}
```

to workaround the issue described in https://github.com/actions/checkout/issues/1467 on builds triggered by tag